chkcryptoboot.conf

```diff
@@ -8,6 +8,9 @@ BOOTMODE=
 #Boot disk (in case of mbr. the disk you GRUB is installed)
 BOOTDISK=
 
+#BIOS boot partition (in case of mbr with GPT. the BOOTDISK partition where GRUB stage 2 is installed)
```

Comment:
core.img, regardless if on post MBR gap or on BIOS BOOT partition is actually stage 1.5, not stage 2. stage 2 is what resides on the actual boot partition. Just a typo fix.

Reply:
Good catch, I'll change that